testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentContainers.java

```diff
@@ -34,4 +38,23 @@ public static boolean isPrestoContainer(String name)
     {
         return name.startsWith(PRESTO);
     }
+
+    /**
+     * Use this method only when you place `tempto-configuration.yaml' in environment configuration directory.
+     */
+    public static void configureTempto(Environment.Builder builder, DockerFiles.ResourceProvider configDir)
+    {
+        builder.configureContainer(TESTS, dockerContainer -> {
+            String environmentName = configDir.getPath("..").toFile().getName();
```

Comment:
Instead of deriving it, can we pass it as an argument.

Reply:
I was done this way after: https://github.com/trinodb/trino/pull/8627#discussion_r700156415

> this filedescription is nice, but it's not copy-paste-friendly. would you consider dropping it?